src/utils/validate-object-id-hook.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const errors = require('feathers-errors');
+const mongoose = require('mongoose');
+
+var validateObjectId = function() {
+  return function(hook) {
+    const id = hook.id;
+    if (id && !mongoose.Types.ObjectId.isValid(id)) {
```

Comment:
The model handles it but it returns 400 BadRequest with `Cast to ObjectId failed`. We need to return 404 NotFound instead. See #18 and #19.